packages/shared/src/dto/workflows/step-data.dto.ts

```diff
@@ -27,6 +29,13 @@ export enum UiComponentEnum {
   IN_APP_SUBJECT = 'IN_APP_PRIMARY_SUBJECT',
   IN_APP_BUTTON_DROPDOWN = 'IN_APP_BUTTON_DROPDOWN',
   URL_TEXT_BOX = 'URL_TEXT_BOX',
+  DIGEST_AMOUNT = 'DIGEST_AMOUNT',
+  DIGEST_UNIT = 'DIGEST_UNIT',
+  DIGEST_KEY = 'DIGEST_KEY',
+  DIGEST_CRON = 'DIGEST_CRON',
+  DELAY_TYPE = 'DELAY_TYPE',
+  DELAY_AMOUNT = 'DELAY_AMOUNT',
+  DELAY_UNIT = 'DELAY_UNIT',
```

Comment:
figma was not loading for me, for now, i created a dedicated value for each component.